scripts/generate_errors.pl

```diff
@@ -48,7 +48,7 @@
                             CAMELLIA CCM CHACHA20 CHACHAPOLY CMAC CTR_DRBG DES
                             ENTROPY ERROR GCM HKDF HMAC_DRBG MD2 MD4 MD5
                             NET OID PADLOCK PBKDF2 PLATFORM POLY1305 RIPEMD160
-                            SHA1 SHA256 SHA512 THREADING XTEA );
+                            SHA1 SHA256 SHA512 SM3 THREADING XTEA );
 my @high_level_modules = qw( CIPHER DHM ECP MD
```

Comment:
As discussed [earlier](https://github.com/ARMmbed/mbedtls/pull/4492#issuecomment-849564319), please edit `tests/scripts/all.sh` and add SM3 every place ALG_MD5 is mentioned.

Reply:
I have checked `all.sh`. it looks good to me.